plugins/nodes/src/js/components/modals/DrainNodeModal.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from "react";
+
+// @ts-ignore
+import { request } from "@dcos/mesos-client";
+import { Trans } from "@lingui/macro";
+import { Modal } from "reactjs-components";
```

Comment:
Have we considered using the ui-kit modal here?

Reply:
I did. We don't use the ui-kit modal in dcos-ui and I figured we should migrate all at once.